lib/atlas/dataset/derived_dataset.rb

```diff
@@ -35,5 +44,21 @@ def scaling_valid
         end
       end
     end
+
+    def init_keys_exist
+      init.each_pair do |key, value|
+        unless InitializerInput.exists?(key)
+          errors.add(:init, "initializer input '#{ key }' does not exist")
```

Comment:
An error message is now gonna read "Init intializer input 'transport_car_oil_share' does not exist".

Maybe `"key '#{ key }' does not exist as initializer input"` gives a nicer result